theano/configdefaults.py

```diff
@@ -430,7 +430,7 @@ def f(suffix=suffix):
         except OSError:
             rc = 1
         if rc != 0:
-            _logger.warning("conda g++ not available, use `conda install m2w64-toolchain`")
+            _logger.warning("g++ not available, if using conda: `conda install m2w64-toolchain`")
```

Comment:
OK, I will change it to look for "Continuum", or anything containing "conda" as that will get both "conda-forge" and "Anaconda".